.tekton/controller-push.yaml

```diff
@@ -20,15 +20,15 @@ spec:
       value: Dockerfile
     - name: infra-deployment-update-script
       value: >
-        sed -i 
+        sed -i
         -e 's|\(https://github.com/redhat-appstudio/jvm-build-service/.*?ref=\)\(.*\)|\1{{ revision }}|'
         -e 's/\(newTag: \).*/\1{{ revision }}/'
         components/hacbs/jvm-build-service/base/kustomization.yaml
 
         ./hack/generate-apiexport-overlays.sh components/hacbs/jvm-build-service/
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio/build-templates-bundle:devel
```

Comment:
do not change controller, it has infra-deployment update - and it's not java project

Reply:
controller no longer changed